tests/ci/docker_images/linux-x86/centos-7_gcc-4x/Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM centos:7
+
+SHELL ["/bin/bash", "-c"]
+
+# This docker image is to test the oldest gcc version we can find
+# on an OS that works with >= JDK10.
+#
+# We manually install go 1.14.10 because of the version of git
+# for CentOS 7 is too old to support the `--unshallow` option
+# https://github.com/golang/go/issues/38373
+RUN set -ex && \
+    yum -y update && \
+    yum --enablerepo=extras install epel-release -y && \
+    yum -y install \
+    gcc \
+    libgcc.i686 \
+    glibc-devel.x86_64 \
+    glibc-devel.i686 \
+    gcc-c++ \
```

Comment:
how can we be assured that this installs the GCC version we intend to test against?